tests/foreman/api/test_host.py

```diff
@@ -299,88 +349,114 @@ def test_positive_create_with_inherited_params(self):
             glob_param_list = {
                 (param.name, param.value) for param in entities.CommonParameter().search()
             }
-        self.assertEqual(len(host.all_parameters), 2 + len(glob_param_list))
+        assert len(host.all_parameters) == 2 + len(glob_param_list)
         innerited_params = {(org_param.name, org_param.value), (loc_param.name, loc_param.value)}
         expected_params = innerited_params.union(glob_param_list)
-        self.assertEqual(
-            expected_params, {(param['name'], param['value']) for param in host.all_parameters}
-        )
+        assert expected_params == {
```

Comment:
Isnt it required to sort both the sides to compare ?

Reply:
no, when you are comparing two dictionaries there is no need to sort them
here is an example:
```
>>> a = {1:'a', 2:'b', 3:'c'}
>>> b = {3:'c', 1:'a', 2:'b'}
>>> a == b
True
```

Reply:
No change.